main.py

```diff
@@ -1,11 +1,36 @@
+class Car:
+    def __init__(self, WheelNumber, PSI, FuelType ):
```

Comment:
ALWAYS in ANY function you write, you should declare what is the 'type' of each argument. 
https://docs.python.org/3/library/typing.html (read only the first section, see the example in their first use case).

It will not be forced by python that the actual arguments sent to this function would be from this type (because python by design is very not limiting and tend to think the programmer knows what he's doing), but it will be easier for the one who uses your function to understand what is does (in this case the function is the Car constructor and the user is yourself, but you should get the habit as if you are writing your code to other people to read and use)

Reply:
Done